stdlib/private/StdlibUnicodeUnittest/StdlibUnicodeUnittest.swift

```diff
@@ -111,7 +112,13 @@ public let normalizationTests: [NormalizationTest] = {
   }
   
   return tests
-}()
+}
+
+// Older version of stdlib with ICU and supporting Unicode 11
```

Comment:
This just means that we haven't kept the tests updated, right? It does not mean that the stdlib in 5.5 only implements Unicode 11, does it?

Reply:
Well, the stdlib in 5.5. is dependent on what ICU is in the system/was in the toolchain. It could be supporting version 11, 12, 13, etc. But yes, this test has not been updated for newer Unicode versions/OS releases that support newer versions.